fixups/FileRedirectionFixup/PathRedirection.cpp

```diff
@@ -28,6 +28,25 @@ struct vfs_folder_mapping
 };
 std::vector<vfs_folder_mapping> g_vfsFolderMappings;
 
+std::filesystem::path CreateDirectoryTree(std::filesystem::path path, std::wstring_view relativePath)

```

Comment:
This logic already exists.  GenerateRedirectedPath in PathRedirection.cpp.  If you pass in true for "ensureDIretoryExists" the method will create the directory tree.

Reply:
Replaced with std::filesystem::create_directories. GenereateRedirectedPath has a subtle difference that it doesn't create the last path fragment.